### DISCUSSION TITLE
Using questions about ChronosBoltPipeline

### DISCUSSION THREAD

**Discussion by yqwang306** [amazon-science/chronos-forecasting#237] [Q&A]
Hello, Thank you very much for your open-source library. I encountered some issues during its usage and would greatly appreciate your help. I noticed that you recently released the Chronos-Bolt version. In this version, if I want to obtain sequence embeddings instead of prediction results, is there a method similar to the **embed** method in ChronosPipeline within ChronosBoltPipeline?

**Comment by abdulfatir** [comment]
Hey @yqwang306, I will open a pull request soon the enable the `embed` functionality for Chronos-Bolt. 

EDIT: Opened #247, you can use that branch for embed functionality until it has been merged into main.

**Reply by abdulfatir** [reply]
@yqwang306 `ChronosBoltPipeline.embed` support has been merged into main. Please install from source.
